### ISSUE TITLE
Divide by zero error if no state.call_candidates

### ISSUE THREAD

**Issue by peterjc** [ikamensh/flynt#37]
I can see how to fix this, but on a project previously flynt'd without the line length setting

```
$ flynt --line_length 99999 project_name/
Running flynt v.0.40

Flynt run has finished. Stats:

Execution time:                            0.401s
Files modified:                            14
Character count reduction:                 2604 (0.98%)

Per expression type:
Old style (`%`) expressions attempted:     86/108 (79.6%)
Traceback (most recent call last):
  File "/Users/xxx/opt/miniconda3/bin/flynt", line 8, in <module>
    sys.exit(main())
  File "/Users/xxx/opt/miniconda3/lib/python3.7/site-packages/flynt/__init__.py", line 78, in main
    transform_concat=args.transform_concats,
  File "/Users/xxx/opt/miniconda3/lib/python3.7/site-packages/flynt/api.py", line 178, in fstringify
    transform_concat=transform_concat,
  File "/Users/xxx/opt/miniconda3/lib/python3.7/site-packages/flynt/api.py", line 105, in fstringify_files
    total_time,
  File "/Users/xxx/opt/miniconda3/lib/python3.7/site-packages/flynt/api.py", line 131, in print_report
    f"`.format(...)` calls attempted:            {state.call_transforms}/"
ZeroDivisionError: division by zero
```

**Comment by ikamensh** [comment]
Thanks for reporting, fixed in 0.40.1

**Comment by peterjc** [comment]
Great - I didn't get round to writing the fix yet
3358ddf5ee5242a17210da64691d9b5a1d261091
